scripts/create_tables.sql

```diff
@@ -1 +1,27 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY,
+  surname VARCHAR,
+  given_name VARCHAR,
+  gender VARCHAR(1),
+  born_on DATE,
+  specialty VARCHAR
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  surname VARCHAR,
+  given_name VARCHAR,
+  gender VARCHAR(1),
+  born_on DATE,
+  insurance VARCHAR
+);
+
+CREATE TABLE appointments (
+  id SERIAL PRIMARY KEY,
+  date TEXT,
```

Comment:
The prompt said feel free to use TEXT for all of the fields, so I did.

"appointments - which should have date, time, and comment columns (feel
free to use the type TEXT for all three) as well as columns referencing
both doctors and patients."

On Wed, Apr 27, 2016 at 9:59 AM, Jason Weeks <notifications@github.com>
wrote:

> In scripts/create_tables.sql
> <https://github.com/ga-wdi-boston/sql-references-diagnostic/pull/70#discussion_r61261492>
> :
>
> > +  born_on DATE,
> > +  specialty VARCHAR
> > +);
> > +
> > +CREATE TABLE patients (
> > +  id SERIAL PRIMARY KEY,
> > +  surname VARCHAR,
> > +  given_name VARCHAR,
> > +  gender VARCHAR(1),
> > +  born_on DATE,
> > +  insurance VARCHAR
> > +);
> > +
> > +CREATE TABLE appointments (
> > +  id SERIAL PRIMARY KEY,
> > +  date TEXT,
>
> why not date?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-boston/sql-references-diagnostic/pull/70/files/5ab71403c26131eb00799a9c33d6ad1b8975498e#r61261492>
>